docs/backend-system/building-backends/08-migrating.md

```diff
@@ -212,8 +212,7 @@ place it in the legacy plugin environment.
 
 ## Cleaning Up the Plugins Folder
 
-For plugins that are private and your own, you can follow a [dedicated migration
-guide](../building-plugins-and-modules/08-migrating.md) as you see fit, at a
+For plugins that are private and your own, you can follow a [dedicated migration guide](../building-plugins-and-modules/08-migrating.md) as you see fit, at a
```

Comment:
Not sure about this one. The microsite doesn't resolve this link properly (as reported on discord) and I wasn't sure why. Trying to not break it across lines to begin with